WORKSPACE

```diff
@@ -39,9 +39,9 @@ load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
 # 3) Check if envoy_build_config/extensions_build_config.bzl is up-to-date.
 # Try to match it with the one in source/extensions and comment out unneeded extensions.
 
-ENVOY_SHA1 = "98c1c9e9a40804b93b074badad1cdf284b47d58b"  # 2021-05-11: v1.18.3
+ENVOY_SHA1 = "9ebffc61cfd2ea11807757b12a4ae7b92ab0dc8c"  # latest envoy commit June9th
```

Comment:
Is this a permanent strategy change that we'll always sync to envoy head? Or it's a one-time thing to fulfill the customer's request? I am not sure the the difference between latest envoy vs released envoy version. Any risks for reliability etc?

Reply:
This is just one-time thing.  The next envoy release is at the end of month.  This customer could not wait for that long.

In general,  we only sync to the released envoy.

Reply:
We had similar discussions in the past about syncing to head vs released envoy. In general we should prefer released envoy because security issues will be easier to patch. Other than security patches, we did not find any other risks in reliability.